[2025-04-14T17:01:10.629+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reddit_pipeline.reddit_extraction scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T17:01:10.649+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reddit_pipeline.reddit_extraction scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T17:01:10.650+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-04-14T17:01:10.677+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): reddit_extraction> on 2025-04-13 00:00:00+00:00
[2025-04-14T17:01:10.693+0000] {standard_task_runner.py:57} INFO - Started process 88 to run task
[2025-04-14T17:01:10.706+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_reddit_pipeline', 'reddit_extraction', 'scheduled__2025-04-13T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpfb1xsgxy']
[2025-04-14T17:01:10.715+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask reddit_extraction
[2025-04-14T17:01:10.790+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_reddit_pipeline.reddit_extraction scheduled__2025-04-13T00:00:00+00:00 [running]> on host 5d379c010d2e
[2025-04-14T17:01:10.847+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Hari Shankar' AIRFLOW_CTX_DAG_ID='etl_reddit_pipeline' AIRFLOW_CTX_TASK_ID='reddit_extraction' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T00:00:00+00:00'
[2025-04-14T17:01:10.863+0000] {logging_mixin.py:151} INFO - connected to reddit!
[2025-04-14T17:01:10.870+0000] {sessions.py:161} WARNING - Retrying due to ConnectionError(MaxRetryError("HTTPSConnectionPool(host='www.reddit.com', port=443): Max retries exceeded with url: /api/v1/access_token (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xffff6ff7fc10>: Failed to establish a new connection: [Errno -2] Name or service not known'))")) status: GET https://oauth.reddit.com/r/dataengineering/top
[2025-04-14T17:01:11.329+0000] {sessions.py:161} WARNING - Retrying due to ConnectionError(MaxRetryError("HTTPSConnectionPool(host='www.reddit.com', port=443): Max retries exceeded with url: /api/v1/access_token (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xffff6fec0340>: Failed to establish a new connection: [Errno -2] Name or service not known'))")) status: GET https://oauth.reddit.com/r/dataengineering/top
[2025-04-14T17:01:14.527+0000] {python.py:194} INFO - Done. Returned value was: /opt/airflow/data/output/reddit_20250414.csv
[2025-04-14T17:01:14.540+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_reddit_pipeline, task_id=reddit_extraction, execution_date=20250413T000000, start_date=20250414T170110, end_date=20250414T170114
[2025-04-14T17:01:14.560+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-14T17:01:14.594+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-14T20:18:34.582+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reddit_pipeline.reddit_extraction scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T20:18:34.587+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reddit_pipeline.reddit_extraction scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T20:18:34.587+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-04-14T20:18:34.593+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): reddit_extraction> on 2025-04-13 00:00:00+00:00
[2025-04-14T20:18:34.595+0000] {standard_task_runner.py:57} INFO - Started process 52 to run task
[2025-04-14T20:18:34.597+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_reddit_pipeline', 'reddit_extraction', 'scheduled__2025-04-13T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpdbey7h6j']
[2025-04-14T20:18:34.599+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask reddit_extraction
[2025-04-14T20:18:34.618+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_reddit_pipeline.reddit_extraction scheduled__2025-04-13T00:00:00+00:00 [running]> on host beccf1ea3c0f
[2025-04-14T20:18:34.646+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Hari Shankar' AIRFLOW_CTX_DAG_ID='etl_reddit_pipeline' AIRFLOW_CTX_TASK_ID='reddit_extraction' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T00:00:00+00:00'
[2025-04-14T20:18:34.728+0000] {logging_mixin.py:151} WARNING - Version 7.7.1 of praw is outdated. Version 7.8.1 was released Friday October 25, 2024.
[2025-04-14T20:18:34.729+0000] {logging_mixin.py:151} INFO - connected to reddit!
[2025-04-14T20:18:35.651+0000] {python.py:194} INFO - Done. Returned value was: /opt/airflow/data/output/reddit_20250414.csv
[2025-04-14T20:18:35.658+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_reddit_pipeline, task_id=reddit_extraction, execution_date=20250413T000000, start_date=20250414T201834, end_date=20250414T201835
[2025-04-14T20:18:35.702+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-14T20:18:35.709+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
